docs/performance/performance_tutorial.md

```diff
@@ -355,6 +355,52 @@ scrape_configs:
 
 Once the environment is setup, you are good to run workloads and collect results. YuniKorn community has some useful tools to run workloads and collect metrics, more details will be published here.
 
+### 1. Scenarios 
+In performance tools, there are three types of tests and feedbacks.
+
+|	test type	|						description						|	diagram	|  		log		|
+| ---------------------	| -----------------------------------------------------------------------------------------------------	| ------------- | ----------------------------- |
+|	e2e test	|	Simulate and record the time in each steps							|	none	|	exist(QPS, timecost)	|
+|	node fairness	|	Monitor node resource usage(allocated/capicity) with lots of pods requests			| 	exist	|	exist			|
+|	thourghput	|	Allocate `pod.spec.starttime` to calculate throughput(pods/sec) with lots of pods requests	|	exist	|	none			|
+
+### 2. Build tool
+Performance tool is in [yunikorn release](https://github.com/apache/incubator-yunikorn-release.git), so clone it to your host. 
+```
+git clone https://github.com/apache/incubator-yunikorn-release.git
+```
+Go to performance tool directory and build it
+```
+cd incubator-yunikorn-release/perf-tools/
+go mod tidy
+go build
+```
+It will look like this.
+![Build-perf-tools](./../assets/perf-tutorial-build.png)
+
+### 3. Set test configuration
+Before start tests, check configuration whether meet your except.
+Default output path is `\tmp`, you can modify `common.outputrootpath` to change it.
+In each scenarios, it contains followings and we can set
+
+|	field			|			description					|
+| ----------------------------- | --------------------------------------------------------------------- |
+|	schedulerNames		|	List of scheduler will run these cases 				|
```

Comment:
you can actually set 2 scheduler names? I wasn't aware of that
I think you need to add some more description to explain this a bit more

Reply:
Yes.
In throughput and fairness cases, this tool can run certain scheduler we require,
It will change scheduler name in pod spec field.
I will explain fields in each case separately in next commit.